### ISSUE TITLE
Playback Resets To 1 Instead of 1.5 For NC Replays on Cutting Edge

### ISSUE THREAD

**Issue by cyperdark** [ppy/osu-stable-issues#1247]
By pressing **5** to "pause" replay and then unpause by pressing **0** reset playback rate to **1** for **NC** replay instead of 1.5 rate
> beatmap: https://osu.ppy.sh/b/2245774
> DT replay: https://osu.ppy.sh/scores/osu/4490217662
> NC replay: https://osu.ppy.sh/scores/osu/4414789150

https://github.com/user-attachments/assets/a95cf636-2367-4124-a44a-20505ef842ca

**Comment by cyperdark** [comment]
hello @peppy can you look at this one, and is there a chance that it could be fixed?

**Comment by peppy** [comment]
I don't think this is important at all. It's also a cutting-edge only feature that is mostly there for us devs.
